Symfony/CS/Tests/Fixer/Contrib/ShortEchoTagFixerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+class ShortEchoTagFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideClosingTagExamples
+     */
+    public function testOneLineFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideClosingTagExamples()
+    {
+        return array(
+            array('<?php echo \'Foo\';', '<?= \'Foo\';'),
+            array('<?php echo \'Foo\'; ?> PLAIN TEXT', '<?= \'Foo\'; ?> PLAIN TEXT'),
+            array('PLAIN TEXT<?php echo \'Foo\'; ?>', 'PLAIN TEXT<?= \'Foo\'; ?>'),
```

Comment:
Please add a test with multiples ``<?=`` tags to ensure that the last one is not missed once you update the collection properly (this is what requires switching to backward iteration). you will need enough occurrences so that the last one moves past the initial length of the token collection after replacements